scripts.js

```diff
@@ -0,0 +1,8 @@
+var femaleNames = ['Asia', 'Kasia', 'Ola', 'Jola'];
+var maleNames = ['Piotrek', 'Marek', 'Arek', 'Jarek'];
+var allNames = femaleNames.concat(maleNames);
+var newName = 'Marian';
+if (allNames.indexOf(newName) === -1) {
+	var x = allNames.push('Marian');
```

Comment:
`var x` is unnecessary :) 

to allNames array you can push name by variable newName